### PR TITLE
Add 'humans' app (previously called ship_pirates)

### PR DESCRIPTION
**Pull request by txels**
Main changes include:
 - Copy contents of previously independent app `ship_pirates`, renamed to `humans`.
 - Settings changes:
   - Add settings that were not on version control (only at the production server).
   - Make secrets into environment variables
   - Split settings into 3 files for clarity: django standard, those from 3rd party apps, and those specific to our project
 - Update of some dependencies that didn't work with the (newer) version of django we use

### CODE REVIEW

File: humans/auth/forms.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+        This allows subclasses to more easily customize the default policies
+        that prevent inactive users from resetting their password.
+        """
+        active_users = User._default_manager.filter(
+            **{
+                '%s__iexact' % User.get_email_field_name(): email,
+                'is_active': True,
+            })
+        import pprint; pprint.pprint(active_users)
```
Comment (darccio) [inline humans/auth/forms.py]: This line can be deleted or replaced by a proper log. I use it to check if an e-mail is registered for a given user (so I knew everything was right when handling new membership requests).

File: shipanaro/settings_django.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+USE_TZ = True
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/1.9/howto/static-files/
+
+STATIC_ROOT = os.path.join(BASE_DIR, "static")
+STATIC_URL = "/static/"
+
+EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+# TODO: prod settings? is this used? the package django-ses in not in Pipfile
```
Comment (darccio) [inline shipanaro/settings_django.py:146]: After setting up Mailbluster as our newsletter manager, we are using Amazon SES as our delivery mail system.

**Review by darccio — APPROVED** [review]
LGTM

File: shipanaro/settings_django.py (hunk 2 above)
Comment (txels) [reply shipanaro/settings_django.py:146]: So we need to add django-ses as a dependency... is this installed on the server directly? Which version?

File: humans/management/commands/check_users.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+from django.core.management.base import BaseCommand
```
Comment (txels) [inline humans/management/commands/check_users.py:1]: This file used to be called `check.py` but it conflicted with core Django command `check`, that is invoked internally when doing `./manage.py test`.

File: humans/models.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        is_new = self.id is None
+        if is_new:
+            self.__ldap__save()
+        password = self._password
+        super().save(*args, **kwargs)
+        if password is not None and not is_new:
+            self.__ldap__set_password(password)
+
+    def __ldap__save(self):
+        # ldap_user = _LDAPUser(LDAPBackend(), username=)
```
Comment (darccio) [inline humans/models.py:18]: @txels This is the code supposed to allow creating users, but I left it commented for some reason. Probably it didn't work or I didn't finish it.

File: shipanaro/settings_django.py (hunk 2 above)
Comment (darccio) [reply shipanaro/settings_django.py:146]: The last one, as we started using Mailbluster this past July.

**Review by darccio — APPROVED** [review]

**Pull request merged**